addons/web_tour/static/src/js/tip.js

```diff
@@ -211,15 +212,17 @@ var Tip = Widget.extend({
             // important to only remove/add the class when necessary to not
             // notify a DOM mutation which could retrigger this function.
             const $oldLocation = this.$el.parent();
-            if (!$location.is($oldLocation)) {
+            if (!$location.is($oldLocation) && !this.tip_opened) {
                 $oldLocation.removeClass('o_tooltip_parent');
                 const cssPosition = $location.css("position");
                 if (cssPosition === "static" || cssPosition === "relative") {
                     $location.addClass("o_tooltip_parent");
                 }
                 this.$el.appendTo($location);
             }
-            this._reposition();
+            if (!this.tip_opened) {
```

Comment:
You check the new condition twice in a row.

Reply:
@qsm-odoo changed.